academic-services/knowledge-exploration-service/reference-get-interpret.md

```diff
@@ -0,0 +1,459 @@
+`complete` | Optional | integer | A value of 1 indicates that interpretations should be generated that infer entity attribute values beyond what was provided in the query. <br/><br/>For example, the for query "microsoft machine le", if `complete` is set to 1 an interpretation could be generated that completes "le" to "learning", i.e. "microsoft machine learning". <br/><br/>A value of 0 indicates that no inference is made and only the exact query text provided is matched. <br/><br/>Defaults to 1.
+`normalize` | Optional | integer | Value of 1 indicates that the query should have normalization rules applied to it before being interpreted. See the [entity schema overview](reference-makes-api-entity-schema.md#normalization-rules) for documentation on the normalization rules that are applied to indexed string fields. <br/><br/>Defaults to 1.
+`offset` | Optional | integer | The number of interpretations to skip in the result set. <br/><br/>Defaults to 0.
+`timeout` | Optional | integer | The maximum amount of time in milliseconds to use when generating interpretations. <br/><br/>If the timeout is hit, all interpretations that have been generated will be returned and a "timedOut" flag will be set in the response indicating that the timeout was hit before the total number of requested interpretations was met. <br/><br/>Defaults to 2000.
+`entityCount` | Optional | integer | The number of entities that should be returned for each interpretation that match the interpreted query expression. <br/><br/>Defaults to 0.
+`attributes` | Optional | string | A list of comma-separated attributes to include for each entity returned for each interpretation. See the [entity schema overview](reference-makes-api-entity-schema.md#entity-types) for the attributes that can be requested. <br/><br/>If an asterisk (*) is specified, all available attributes will be returned. <br/><br/>Defaults to an empty string.
+
+## Responses
+
+Name | Type | Description
+--- | --- | ---
+200 OK | [InterpretResponse](#interpretresponse) | Interpret response successfully generated and returned.
+
+## Definitions
+
+| | |
+| --- | --- |
+[InterpretResponse](#interpretresponse) | Interpret response information.
+[Interpretation](#interpretation) | Information for an individual query interpretation.
+[InterpretationRuleMatch](#interpretationrulematch) | Information returned for grammar rules matched during interpretation.
+[InterpretationRuleMatchOutput](#interpretationrulematchoutput) | Output generated for a grammar rule match.
+
+### InterpretResponse
+
+Name | Type | Description
+--- | --- | ---
+interpretations | [Interpretation](#interpretation)[] | Array of interpretations ordered by relevance.
+timed_out_count | integer | The number of index partitions that timed out before returning the requested number of interpretations.
+string | The natural language query used to generate interpretations.
```

Comment:
There should be 4 entries (according to the sample response)
- query
- interpretations
- timeout_out_count
- tme_out